content/reviewers_response.md

```diff
@@ -0,0 +1,642 @@
+
+**We are using cosine similarity within our linguistic variation analysis.
+Regarding our missing preprint publish link analysis, pure euclidean distance has the same performance as cosing similarity.
+Under this task, preprints and their published counterparts have significantly lower distances regardless of the distance metric used (comparison figure provided below).**
+
+![Distance comparison](https://github.com/greenelab/annorxiver/blob/ff456c5c5cb7cddd16400f6e5b74422e1b33d349/biorxiv/article_distances/output/figures/distance_metrics_top_k_precision.png)
+
+> Similarly, I was wondering how the "journal-based" approach, which the authors mention briefly against the influence of high publication frequency journals, was implemented. Further, if it could have been avoided by avoiding the Euclidean space.
+>
+> The mapping of similarity seems to be based on individual pairs of text and as such it would seem vulnerable of shifting distributions (e.g.: if published articles were somewhat different from preprints, as implied in Figure 1A). I would suspect that the authors would be able to improve their performance even further by doing global matching between many pairs (… again see their adherence to a weaker approach as something that ultimately strengthens their findings). Again, a comment on the rationale of their chosen approach could convey additional non-evident considerations.
+
+**Our journal-based approach is still required as cosine/euclidean distance does not combat the issue of journal overrepresentation.
+We control for this issue by calculating a centroid for each journal.
+This centroid is constructed by taking the average of every published document present within each journal.
+This way, every journal will have an equal representation for our classifier.
+The shift in distribution could affect our paper-paper classifier which is why we provided top 10 results instead of best paper.
+Another way to combat this issue is to use a technique called orthogonal procrustes, which is a linear algebra techique designed to align two vector spaces.**
+
+> I love the web application!
+
+**We are happy you like the application.**
+
+> No statistics are given for the enrichments in Figure 1B-E.
+
+**We report each token with a 95% confidence interval (CI). 
+Our frequency counts are sufficiently large that all calculated p-values are approximately or exactly zero.**
+
+> I would welcome a supplemental analysis, that removes single letters and special characters from the analysis of Figure 1B-E as they might change the baseline.
+
+**We have added this analysis to our supplement.**
```

Comment:
Can you put the figure here too? And also provide the reviewer with the figure/section in the supplement? Make their life easy! :-D